### PR TITLE
CLDSRV-598: Add overhreadField to metadata-only operations

### PR DESCRIPTION
**Pull request by dvasilas**
Metadata-only operations (objectPutRetention, objectPutLegalHold, objectPutTagging, objectPutACL) overwrite the metadata of an object.

These operations currently do not attach the overheadField to metadata put operations.

As a result, the entries in the metadata op log for those operations do not include overhead information, and scuba interprets them as object puts, resulting in wrong metric updates.

Issue: CLDSRV-598

### CODE REVIEW

**Comment by dvasilas** [comment]
ping

File: lib/api/objectPutACL.js
Comment (ghost) [inline lib/api/objectPutACL.js:1]: > As a result, the entries in the metadata op log for those operations do not include overhead information, and scuba interprets them as object puts, resulting in wrong metric updates.

We may want to also review the metadata operations done in backbeat or route backbeat here, just to make sure they doesn't lead to wrong behavior in scuba

Comment (ghost) [reply lib/api/objectPutACL.js:1]: Also, don't we have the same issue with metadata-only deletes, that also alter the metadata?

Comment (dvasilas) [reply lib/api/objectPutACL.js:1]: > We may want to also review the metadata operations done in backbeat or route backbeat here, just to make sure they doesn't lead to wrong behavior in scuba.

I created https://scality.atlassian.net/browse/S3C-9618 for that.

> Also, don't we have the same issue with metadata-only deletes, that also alter the metadata?

Good point ! 
I see `objectDeleteTagging` that should be fixed. Is there another one that I am missing ?

Comment (ghost) [reply lib/api/objectPutACL.js:1]: I don't see anything else, but maybe @Kerkesni or @francoisferrand will have some

**Comment by dvasilas** [comment]
I am closing this because this is not the correct fix for the issue. 
The fix will be handled by CLDSRV-552

**Pull request closed without merging**